### ISSUE TITLE
Progress bar for verification

### ISSUE THREAD

**Issue by tomato42** [tomato42/ecpp-verifier#15]
Because verifying even a modest certificate (1536 bit prime) takes good half a minute and can take more than an hour for big primes (8196 bit), it would be nice to have a progress bar or completion percentage reported

**Comment by tomato42** [comment]
after #25 (especially if used together with gmpy2), the verification of even 8192 bit primes takes few minutes, so per-certificate progress
notification is less important now (though it would still be useful)
